swisstopo_geoengine/geo_view/geo_raster_layer.py

```diff
@@ -10,4 +10,4 @@ class GeoRasterLayer(models.Model):
     raster_type = fields.Selection(selection_add=[('swisstopo', 'Swisstopo')])
     layername = fields.Char('Layer Machine Name')
     time = fields.Char('Time Dimension')
-    projection = fields.Char('Projection')
+    # projection = fields.Char('Projection')
```

Comment:
Please remove it completely instead of commenting it. The change is quite simple and thus it has very few value to keep it.